query-old/docs/api/created-api/hooks.md

```diff
@@ -91,18 +91,40 @@ The query arg is used as a cache key. Changing the query arg will tell the hook
 #### Signature
 
 ```ts
-type MutationHook = () => [
-  (
-    arg: any
-  ) => {
-    unwrap: () => Promise<ResultTypeFrom<D>>;
-  },
-  MutationSubState<D> & RequestStatusFlags
-];
+type UseMutation<Definition> = () => [UseMutationTrigger<Definition>, UseMutationResult<Definition>];
```

Comment:
I don't know the best way to write this, but like this it is omitting `selectFromResult`

Reply:
I've added in a pseudo-type for `UseMutationStateOptions`, but it's definitely getting very crowded. What are your thoughts on removing the `Definition` pseudo-generic from this section completely? My thoughts are that it's already an inaccurate generic compared to the real types, and it probably clutters the explanation more than it clarifies anything.